libsolidity/formal/CHC.cpp

```diff
@@ -43,53 +43,31 @@ using namespace solidity::frontend;
 CHC::CHC(
 	smt::EncodingContext& _context,
 	ErrorReporter& _errorReporter,
-	map<util::h256, string> const& _smtlib2Responses,
-	ReadCallback::Callback const& _smtCallback,
+	[[maybe_unused]] map<util::h256, string> const& _smtlib2Responses,
+	[[maybe_unused]] ReadCallback::Callback const& _smtCallback,
 	[[maybe_unused]] smtutil::SMTSolverChoice _enabledSolvers
```

Comment:
I think this one is always used now, so you can remove `[[maybe_unused]]`
```suggestion
	smtutil::SMTSolverChoice _enabledSolvers
```

Reply:
True